### ISSUE TITLE
Error establishing a database connection & Localhost links

### ISSUE THREAD

**Issue by wuffo** [soderlind/vscode-remote-try-wp#3]
Getting the following in the log. Plus, Wordpress links go to localhost [i.e. Wordrpress opens in the browser at the .dev URL, but at the home page, all links are then localhost.]. Kinda new at this - possible it's an easy fix. Gotta read more.... Thanks for any pointers.

```
up to date, audited 1 package in 273ms

found 0 vulnerabilities

> vscode-remote-try-wp@0.0.1 build
> echo "No build script specified" && exit 0

No build script specified
Setting up WordPress at https://wuffo-vscode-remote-try-wp-9v4962gv4-8080.githubpreview.dev
Error: Failed to get current SQL modes. Reason: ERROR 2002 (HY000): Can't connect to MySQL server on 'db' (115)

Error: Error establishing a database connection. This either means that the username and password information in your `wp-config.php` file is incorrect or we can’t contact the database server at `db`. This could mean your host’s database server is down.
Activate vscode-remote-try-wp
Error: Error establishing a database connection.
le '/var/www/html/xdebug.log' could not be opened.

```

**Comment by soderlind** [comment]
Before I dive into this, kind of busy at work, could you please fork https://github.com/soderlind/super-admin-all-sites-menu and see if you're able to [run it in codepaces](https://github.com/soderlind/super-admin-all-sites-menu#in-github-codespaces)?

**Comment by wuffo** [comment]
Yes, seems to run fine. But Site Health reports:

1. A scheduled event has failed: The scheduled event, wp_https_detection, failed to run. Your site still works, but this may indicate that scheduling posts or automated updates may not work as intended.
2. The REST API did not behave correctly: The REST API is one way WordPress, and other applications, communicate with the server. One example is the block editor screen, which relies on this to display, and save, your posts and pages. The REST API did not process the context query parameter correctly.

**Comment by soderlind** [comment]
Ref 2: The latest version, 1.6.4, should have fixed this. Please look in wp-admin->Plugins and check if you're running 1.6.4

**Comment by wuffo** [comment]
Yeah, the latest version of the plugin was installed, as well as WP 6.0. Removing the plugin didn't affect the errors.
